src/utils/viewer/viewer.cpp

```diff
@@ -515,9 +587,15 @@ void Viewer::keyPressEvent(KeyEvent& event) {
   }
   renderCamera_->node().setTransformation(
       rgbSensorNode_->absoluteTransformation());
+
   redraw();
 }
 
+void Viewer::updateText(float fps) {
+  // update the text occasionally
+  performanceText_->render(Cr::Utility::formatString("FPS: {:.1f}", fps));
```

Comment:
@mosra:
I do not see the description of this function in your doc, btw.
https://doc.magnum.graphics/magnum/classMagnum_1_1Text_1_1Renderer.html#pub-static-methods

Reply:
Oh, never mind. It is here:
https://doc.magnum.graphics/magnum/classMagnum_1_1Text_1_1AbstractRenderer.html#a1730154b16f3884ef10b29b7582a759b